act-rules-format.bs

```diff
@@ -245,183 +259,212 @@ For other aspects that are not well defined, an ACT Rule MUST include either a d
 
 <div class=example>
-  <p><strong>Example</strong>: The applicability of an atomic rule testing <a href="https://www.w3.org/WAI/WCAG21/quickref/#audio-control" target="_blank">WCAG 2.1 Audio Control:</a></p>
+  <p>**Example:** The applicability of an atomic rule testing [WCAG 2.1 Audio Control](https://www.w3.org/WAI/WCAG21/quickref/#audio-control):</a></p>
   <blockquote>
-    <p>Any <code>video</code> or <code>audio</code> element(s) with the <code>autoplay</code> attribute, as well as any <code>object</code> element(s) that is used for automatically playing video or audio when the web page loads.</p>
+    <p>Each `video` or `audio` element with the `autoplay` attribute, as well as each `object` element that is used for automatically playing video or audio when the web page loads.</p>
+    <p>**Note: A web page is considerd "loaded" when the `document.readySate` is set to `complete`.</p>
   </blockquote>
 </div>
 
 
-Applicability for Composite Rules {#applicability-composite}
-----------------------------------------------------------
+### Applicability for Composite Rules ### {#applicability-composite}
 
-A [=composite rule=] defines how the [=outcomes=] from rules in its [input rules list](#input-rules-list) are used to determine a single outcome.
+A [=composite rule=] defines how the [=findings=] from rules in its [input rules](#input-rules) are used to determine a single [=outcome=].
 
-The applicability of a composite rule is defined as the union of all the applicability sections of its [=atomic rules=] as well as atomic rules that make up any composite rules in the [input rules list](#input-rules-list). Rule authors MAY describe the applicability for composite rules. This can be useful if it is difficult to express the combined applicability in plain language. If the composite rule includes applicability, it MUST be the union of all the applicability sections of the atomic rules.
+The applicability of a composite rule is defined as the union of all the applicability sections of rules in the [input rules](#input-rules). This includes [=atomic rules=], and [=composite rules=] which themselves have a union applicability. Rule authors MAY ommit a description of the applicability for composite rules. This can be useful if it is difficult to express the combined applicability in plain language. If the composite rule includes applicability, it MUST be the union of all the applicability in the [input rules](#input-rules).
```

Comment:
Editorials needed for "it MUST be the union of all the applicability in the [input rules]". Maybe add "sections" to "applicability"

Reply:
I think the issue is with the "the" in the sentence. How about "all applicability definitions" instead of "sections"? The grammar of the sentence does seem wrong to me too.